src/main/java/org/wso2/carbon/salesforce/connector/QueryMoreIterator.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change the license headers since we are creating a new version of the files?